diff --git a/defs.bzl b/defs.bzl
--- a/defs.bzl
+++ b/defs.bzl
@@ -28,3 +28,14 @@
+ARM_LINUX_RUSTC_LINK_FLAGS = select({
+    "@platforms//cpu:aarch64": select({
+        "@platforms//os:linux": [
+            "-C",
+            "link-arg=-lgcc",
+        ],
+        "//conditions:default": [],
+    }),
+    "//conditions:default": [],
+})
+
 WINDOWS_GNULLVM_INCOMPATIBLE = select({
     "@llvm//constraints/windows/abi:gnullvm": ["@platforms//:incompatible"],
     "//conditions:default": [],
